beast/tools/read_beast_data.py

```diff
@@ -111,19 +111,23 @@ def read_noise_data(
 def read_sed_data(
     filename,
     param_list=['Av', 'Rv', 'f_A', 'M_ini', 'logA', 'Z', 'distance'],
+    return_params=False,
 ):
     """
     Read in the beast data needed by all the pixels
 
     Parameters
     ----------
     filename : string
-       name of the file with the BEAST physicsmodel grid
+        name of the file with the BEAST physicsmodel grid
 
     param_list : list of strings
-       the set of parameters to extract
-       default = [Av, Rv, f_A, M_ini, logA, Z, distance]
-       If set to None, return the list of possible parameters
+        The set of parameters to extract (default: Av, Rv, f_A, M_ini, logA, Z,
+        distance).  If set to 'all', extract all parameters in the grid.
+
+    return_params : boolean (default=False)
+        If True, return the list of all parameters in the grid.  Useful for
+        checking what columns are present.
```

Comment:
This is still not clear to me.  If this keyword is set, then all the parameters *and* the seds and lams are returned.  This sounds like more than just all the parameters.

Reply:
`seds` and `lamb` are also stored in the model grid, they're just in different branches (is that the right word?) of the hdf5 file.  So it was easier to manually add them to the list of parameters, rather than formulaically add them in the same way as the other parameters.

Reply:
But they are not parameters.  So stating that all parameters are returned is a bit misleading as *more* than all the parameters are returned.

Reply:
Ah, I think I understand now.  Would a better phrasing be "parameters and model fluxes"?

Reply:
Yes.